charts/crowdsec/Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.17.1
+version: 0.17.2
```

Comment:
Don't bump the versions in PRs, we do it ourselves when we create a new release.